catalog/dags/data_refresh/distributed_reindex.py

```diff
@@ -27,6 +25,10 @@
     PRODUCTION,
     Environment,
 )
+from common.operators.TemplatedConnectionHttpOperator import (
+    TemplatedConnectionHttpOperator,
+)
+from common.sensors.TemplatedConnectionHttpSensor import TemplatedConnectionHttpSensor
```

Comment:
Tiniest nit: It seems a bit wonky to have a `CapsCase` module import with then a `CapsCase` class import 😅 Do you think we could change these to the following?

```suggestion
from common.operators.http import TemplatedConnectionHttpOperator
from common.sensors.http import TemplatedConnectionHttpSensor
```

Reply:
Ah, did this on autopilot and now you point it out I hate it! Strong agree, fixing immediately :)